appveyor.yml

```diff
@@ -9,23 +9,26 @@ environment:
   - platform: x86
     APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
     BUILD_SHARED_LIBS: OFF
+    CMAKE_GENERATOR: Ninja
 
 # VS 2017
-#  - platform: x64
-#    APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-#    BUILD_SHARED_LIBS: ON
+  - platform: x64
```

Comment:
Appveyor tends to be a contention point since we have just one single concurrent instance shared by OSGeo projects. We should limit the work done on it. Basically the only point of it compared to github actions is that it has VS2015. If we have VS2015 in appveyor and VS2019 in github actoins, we can hope for VS2017 to be OK

Reply:
Good to know, I'll limit this to one appveyor job and try to leverage more github actions.